src/runtime/link-accounts-flow.js

```diff
@@ -196,17 +175,37 @@ export class LinkCompleteFlow {
    * @return {!Promise}
    */
   start() {
-    return this.activityIframeViewPromise_.then((activityIframeView) => {
-      this.activityIframeView_ = activityIframeView;
+    if (this.response_['saveAndRefresh']) {
+      this.complete_(this.response_, this.response_['linked']);
+      return Promise.resolve();
+    }
+
+    return this.clientConfigManager_.getClientConfig().then((clientConfig) => {
+      const index = this.response_['index'] || '0';
+      this.activityIframeView_ = new ActivityIframeView(
+        this.win_,
+        this.activityPorts_,
+        feUrl(
+          '/linkconfirmiframe',
+          {},
+          clientConfig.usePrefixedHostPath,
+          'u/' + index
+        ),
+        feArgs({
+          'productId': this.deps_.pageConfig().getProductId(),
+          'publicationId': this.deps_.pageConfig().getPublicationId(),
+        }),
+        /* shouldFadeBody */ true
+      );
 
       const promise = this.activityIframeView_.acceptResultAndVerify(
         feOrigin(),
         /* requireOriginVerified */ true,
         /* requireSecureChannel */ true
       );
       promise
-        .then((response) => {
-          this.complete_(response);
+        .then((response = {}) => {
+          this.complete_(response, response['success'] || false);
```

Comment:
```suggestion
          this.complete_(response, !!response['success']);
```

Reply:
Done. I submitted the fix locally. I've seen weird CLA issues pop up with commit suggestions through GH. I've had to back out and modify commit messages just to resolve them 🤦